docs/api-summary.md

```diff
@@ -23,5 +23,5 @@ The API is secured using an API Key.
 |PUT       |/api/templates/{file}      |Stores a template in the template store          |
 |DELETE    |/api/templates/{file}      |Deletes a template                               |
 |POST      |/api/UpdateBaseTemplates   |Updates base templates (deletes existing data). This should be used only when latest version of templates needs to be pulled.
-|POST      |/api/convert/{srcDataType} |Takes data, and temporary templates as input and outputs FHIR data after applying the templates on the data. The entry-point template is passed base64-encoded in templateBase64 parameter, whereas other overriding templates are passed in the templatesOverrideBase64 parameter.  templatesOverrideBase64 is a base64-encoded json object containing map between the template name and the template content.|
-|POST      |/api/convert/{template}    |Takes data and converts to FHIR using the {template} that is stored on the server.|
+|POST      |/api/convert/{srcDataType} |Takes data, and temporary templates as input and outputs FHIR data after applying the templates on the data. The entry-point template is passed base64-encoded in templateBase64 parameter, whereas other overriding templates are passed in the templatesOverrideBase64 parameter.  templatesOverrideBase64 is a base64-encoded json object containing map between the template name and the template content. You can also set `unusedSegments` and `invalidAccess` parameters in the query string to fetch UI-related segments.|
```

Comment:
Will we request unusesSegments or invalidAccess independently? If not, we can combine them together into one option like "traceInfo=true".

Reply:
The good point is that if we are going to add more trace or debug information in the future, we do not need to more "xxx=true" segments in the URL, and we can update the names of existing fields flexibly.

Reply:
@ranvijaykumar What kind of implementation do you think is more appropriate? Use one parameter(like `traceInfo`) or multiple separate parameters(like `unusedSegments` and `invalidAccess` and so on)?

Reply:
unusedSegment is useful during template editing, while invalidAccess can be useful in production. Let's keep these separate.

Reply:
Got it. Thanks!